docs/_articles/en/apex/writing.md

```diff
@@ -5,11 +5,11 @@ lang: en
 
 Use the Apex extension for VS Code to access code-editing features such as code completion, go to definition, view outlines of Apex classes and triggers, refactoring, and find syntactic errors in your code.
 
-## Code-Completion
+## Code Completion
```

Comment:
I changed these because it isn't normal to use them hyphenated even if it might be technically correct. See: https://code.visualstudio.com/docs/editor/intellisense

Reply:
Yeah, I think the dehyphenation is just fine.